contrib/podmanimage/stable/containers.conf

```diff
@@ -0,0 +1,11 @@
+[containers]
+netns="host"
+userns="host"
+ipcns="host"
+utsns="host"
+cgroupns="host"
+cgroups="disabled"
+[engine]
+cgroup_manager = "cgroupfs"
```

Comment:
This line was under `[containers]` in Buildah.  Should it be in there or here?

Reply:
This is an engine field.  It should be here.  Buildah does not use, I believe.

Reply:
Actually we do need it to prevent the warning.